scripts/setup.sh

```
@@ -2,7 +2,7 @@
 
 ENV_FILE=".env"
 
-ENV_HRM_API="./hrm-api/.env"
+ENV_E_COMMERCE_API="./e-commerce-api/.env"
```

Shellcheck (0.11.0)

[warning] ENV_E_COMMERCE_API appears unused. Verify use (or export if used externally).

(SC2034)